app/views/users/new.html.erb

```diff
@@ -86,18 +86,223 @@
+    l’Événement et à observer les consignes en cas d’urgence;
+  • accepte de suivre les procédures de l’Université, de signaler tout incident
+    dont il est témoin et de respecter les exigences en matière de santé et de
+    sécurité sur les terrains de l’Université ou ailleurs pendant sa participation
+    à l’Événement;
+  • consent à n’entreprendre aucun processus ni aucune procédure ou activité dont
+    il n’a pas déjà discuté avec le responsable de l’Événement sans avoir d’abord
+    reçu à cet égard de la formation, des instructions ou un encadrement du
+    surveillant compétent désigné;
+  • comprend et accepte pleinement que, s’il se livre à toute activité ne faisant
+    pas partie de l’Événement planifié, il est entièrement responsable des
+    conséquences de sa conduite;
+  • comprend et accepte pleinement que, s’il déroge à des règles ou conditions
+    fixées au cours de l’Événement, on pourrait lui demander de quitter les lieux.
+</br>
+
+<b>JE RENONCE À TOUTE REVENDICATION</b> que je pourrais vouloir formuler par suite
+de ma participation à l’Événement contre l’Université d’Ottawa, son Bureau des
+gouverneurs ou ses administrateurs, dirigeants, employés, mandataires,
+entrepreneurs indépendants, sous-traitants, représentants, successeurs et ayants
+droit, ou contre les instructeurs, entraîneurs, gestionnaires, bénévoles,
+commanditaires, agents et dirigeants qui sont associés ou liés de quelque
+manière que ce soit à l’Événement (ci-après appelés collectivement
+« Renonciataires »). De plus, <b>JE DÉGAGE LES RENONCIATAIRES </b> de toute
+responsabilité relative aux pertes, dommages, décès, préjudices ou dépenses dont
+moi ou mon plus proche parent pourrions devoir assumer les conséquences par suite
+de ma participation à l’Événement, QUELLE QU’EN SOIT LA CAUSE, PAR EXEMPLE UNE
+NÉGLIGENCE, UNE VIOLATION DE CONTRAT OU UN MANQUEMENT À UN DEVOIR DE DILIGENCE
+PRÉVU OU NON PAR LA LOI — NOTAMMENT LA <b><i>LOI SUR LA RESPONSABILITÉ DES</i></b>—
+COMMIS PAR LES RENONCIATAIRES, Y COMPRIS LE DÉFAUT DE ME PROTÉGER OU DE PRENDRE
+DES MESURES RAISONNABLES POUR ME PROTÉGER CONTRE LES RISQUES ET DANGERS
+SUSMENTIONNÉS.
+
+<b>J’ACCEPTE</b>, à la demande de l’Université, de quitter l’Événement si je ne
+respecte pas les instructions ou directives de l’Université, ou si je commets une
+infraction en matière de santé et de sécurité.
+<b>
+ON M’A RECOMMANDÉ ET DONNÉ L’OCCASION D’OBTENIR DES CONSEILS D’UN JURISTE IMPARTIAL
+AVANT DE SIGNER LE PRÉSENT DOCUMENT.</b>
+
+En cochant cette case, <b>J’ACCEPTE ET J’ASSUME PLEINEMENT</b> tous les risques et
+dangers pour la santé et la sécurité que peut comporter ma participation à
+l’Événement.
+</pre>
+                   <%= content_tag :span, @new_user.errors[:read_and_accepted_waiver_form].first, class: 'form-error' %>
+                   <%= f.radio_button :read_and_accepted_waiver_form, true, :onclick => "showCreateButton()", style:"display:inline;width:15px;"%>
```

Comment:
doesn't having a checkbox make more sense than a radio button? cus if the person un-clicks this, the sign up button toggles but the radio button stays checked. 

Also, isn't it better to use enable, disable for the button instead of having two buttons? (This is just a thought I have to try it and see if it works. We'll talk about it)